BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTests.cs

```diff
@@ -1,4 +1,12 @@
-﻿using BricksAndHearts.ViewModels;
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using BricksAndHearts.Controllers;
+using BricksAndHearts.Database;
+using BricksAndHearts.Services;

```

Comment:
I don't think any of these are being used... could you tidy up unused imports? `CTRL + ALT + O` should do it. or you can just manually delete the grayed out lines in Rider